Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs

```diff
@@ -126,10 +125,10 @@ public void RegisterOnPremiseTarget(string key, Uri baseUri)
                 await Start();
                 _logger.Info("Connected to relay server. #" + _id);
             }
-            catch (Exception ex)
+            catch (Exception)
             {
                 _logger.Info("***** ERROR WHILE CONNECTING: #" + _id);
-                Delay(5000).ContinueWith(_ => Start());
+                await Delay(5000).ContinueWith(_ => Start());
```

Comment:
@ManuelRauber I'd guess that await should not be necessary here. 

(All other changes would be fine by me and seem more than reasonable ... :))